site2/docs/client-libraries-java.md

```diff
@@ -675,6 +676,7 @@ String|`topicName`|Topic name. |None
 int|`receiverQueueSize`|Size of a consumer's receiver queue.<br/><br/>For example, the number of messages that can be accumulated by a consumer before an application calls `Receive`.<br/><br/>A value higher than the default value increases consumer throughput, though at the expense of more memory utilization.|1000
 ReaderListener&lt;T&gt;|`readerListener`|A listener that is called for message received.|None
 String|`readerName`|Read name.|null
+String| `subscriptionName`|Subscription name|None
```

Comment:
The default value is not 0
When there is a single topic, the default subscription name is: "reader-" + 10-digit UUID
When there are multiple topics, the default subscription name is: "The default subscription name is: "reader-" + 10-digit UUID-" + 10-digit UUID

Reply:
thank you for your comment.